setup.py

```diff
@@ -130,11 +108,10 @@
     "keras>2.9,<2.16",
     "keras-nlp>=0.3.1,<0.14.0",  # keras-nlp 0.14 doesn't support keras 2, see pin on keras.
     "librosa",
-    "natten>=0.14.6,<0.15.0",
```

Comment:
The pin on GitPython was added to fix CI (https://github.com/huggingface/transformers/pull/28432) -- removing to test whether it is still needed